MapleServer2/Packets/ResponseCubePacket.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+using MapleServer2.Tools;
+using MapleServer2.Servers.Game;
+using MapleServer2.Data.Static;
+using Maple2Storage.Types.Metadata;
+
+namespace MapleServer2.Packets
+{
+    public static class ResponseCubePacket
+    {
+        public static Packet Pickup(GameSession session, byte[] coords)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.RESPONSE_CUBE);
+
+            // Convert to signed byte array
+            sbyte[] sCoords = Array.ConvertAll(coords, b => unchecked((sbyte) b));
+            // Default to rainbow tree
+            int weaponId = 18000004;
```

Comment:
This line of code, and also the code below it finding the `mapObject` should also move to the handler, and have the packet method just get fed the final results

Reply:
Can't be moved because of `sbyte[]` issue as stated previously.